hack/e2e-patch-codecov.sh

```diff
@@ -68,42 +63,6 @@ cat >> $CONTROLLER_MANAGER_DEPLOYMENT << EOF
 {{- end }}
 EOF
 
-# for SCHEDULER_DEPLOYMENT, no `env:` added with default values.
-line=$(grep -n 'name: kube-scheduler' $SCHEDULER_DEPLOYMENT | cut -d ":" -f 1)
-head -n $(($line-1)) $SCHEDULER_DEPLOYMENT > $TMP_SCHEDULER_DEPLOYMENT
-cat >> $TMP_SCHEDULER_DEPLOYMENT <<EOF
-        env:
-        - name: COMPONENT
-          value: "scheduler"
-        volumeMounts:
-          - mountPath: /coverage
-            name: coverage
-EOF
-tail -n +$line $SCHEDULER_DEPLOYMENT >> $TMP_SCHEDULER_DEPLOYMENT
-sed -i 's/^{{\- end }}$//g' $TMP_SCHEDULER_DEPLOYMENT
-cat >> $TMP_SCHEDULER_DEPLOYMENT << EOF
-      volumes:
-        - name: coverage
-          hostPath:
-            path: /mnt/disks/coverage
-            type: Directory
-{{- end }}
-EOF
-mv -f $TMP_SCHEDULER_DEPLOYMENT $SCHEDULER_DEPLOYMENT
-
-cat >> $DISCOVERY_DEPLOYMENT << EOF
-          - name: COMPONENT
-            value: "discovery"
-        volumeMounts:
-          - mountPath: /coverage
-            name: coverage
-      volumes:
-        - name: coverage
-          hostPath:
-            path: /mnt/disks/coverage
-            type: Directory
```

Comment:
This will cause two `volumes` to exist after patched if the K8s version >=1.19.0, see #4202.

As tidb-scheduler is no longer recommended, removing this code coverage path should be OK.